docs/getting_started.rst

```diff
@@ -0,0 +1,49 @@
+===============
+Getting Started
+===============
+
+:doc:`Wagtail Autocomplete <index>` provides an edit handler that allows an editor to
+select related objects via a quick autocompleted searching interface.
+
+Requirements
+============
+
+Wagtail Autocomplete requires Wagtail version 1.11 or newer.
+
+.. warning::
+
+    We do not currently support Wagtail 2.x, but we're working on it!
+
+Installation
+============
+
+Install with ``pip``:
+
+.. code-block:: sh
+
+    pip install https://github.com/emilyhorsman/wagtail-autocomplete/tarball/master/
```

Comment:
Could you upload it to PyPI?

Reply:
Yes! That's the plan, but it'll be on a future branch. Tracking in #18